Sources/BraveShared/BraveStrings.swift

```diff
@@ -2457,9 +2457,9 @@ extension Strings {
         comment: "Table cell title for cell that shows when the VPN subscription expires.")
 
     public static let settingsManageSubscription =
-      NSLocalizedString("vpn.settingsManageSubscription", tableName: "BraveShared", bundle: .strings,
-        value: "Manage Subscription",
-        comment: "Button to manage your VPN subscription")
+          NSLocalizedString("vpn.settingsManageSubscription", tableName: "BraveShared", bundle: .strings,
+          value: "Manage Subscription",
+          comment: "Button to manage your VPN subscription")
```

Comment:
The indentation still wrong tho

Reply:
will fix, thanks, xcode indents it completely different compare to other strings here